apolloschurchapp/src/tabs/index.js

```diff
@@ -96,6 +105,11 @@ const HomeTab = createFeatureFeedTab({
     headerCenter: HeaderCenter,
     headerLeft: HeaderLeft,
     headerLargeTitle: false,
+    headerStyle: {
+      blurEffect: 'systemChromeMaterial',
```

Comment:
Cool! Does this work on Android too?

Reply:
unfortunately it's iOS only. But android has different header styles I believe